### ISSUE TITLE
Compatibility with Video.js v8

### ISSUE THREAD

**Issue by phloxic** [videojs/generator-videojs-plugin#317]
Is this generator still the recommended way for a plugin project? Are there plans to make it compatible with Video.js v8?

Currently it fails to build working plugins, see for instance https://github.com/phloxic/videojs-sprite-thumbnails/issues/45

I strongly suspect it's because it creates es5 compatible builds, which, apart from breaking, does not make sense with Video.js v8.

**Comment by misteroneill** [comment]
It is the recommended way to create a plugin project, but we haven't had time to update it. We are planning to do so in the next month or so, though!

**Comment by phloxic** [comment]
Thanks for the confirmation @misteroneill. I thought as much, just wanted to make sure I don't wait in vain.

**Comment by phloxic** [comment]
@misteroneill - FWIW it looks I can build an es6 plugin by nicking @mr-ben's [tip](https://github.com/videojs/video.js/issues/8170#issuecomment-1480370207) and changing the browserslist in package-json to:
```json
"browserslist": [
  ">0.2%",
  "not dead",
  "not op_mini all",
  "not safari < 10",
  "not chrome < 51",
  "not android < 5",
  "not ie < 12"
],
```

**Comment by phloxic** [comment]
This seems to be solved in v9.0.0 without extra gymnastics. Thank you.

Probably for Video.js v7.x one should stick with v8.0.0 of the generator instead of fiddling with the version requirement in package.json?
